app/models/case/sar/offender.rb

```diff
@@ -72,11 +73,11 @@ def searchable_fields_and_ranks
   has_many :data_requests, dependent: :destroy, foreign_key: :case_id
   accepts_nested_attributes_for :data_requests
 
-  validates :third_party, inclusion: { in: [true, false], message: "can't be blank" }
+  validates :third_party,          inclusion: { in: [true, false], message: "can't be blank" }
   validates :flag_as_high_profile, inclusion: { in: [true, false], message: "can't be blank" }
 
-  validates :name, presence: true, if: -> { third_party }
   validates :third_party_relationship, presence: true, if: -> { third_party }
+  # validates :third_party_company_name, presence: true, if: -> { third_party }
```

Comment:
shall we remove the commented out line here?

Reply:
Oh yes! Doh!